diff --git a/envs/staging/email/main.tf b/envs/staging/email/main.tf
index 5efab9c..8181c91 100644
--- a/envs/staging/email/main.tf
+++ b/envs/staging/email/main.tf
@@ -40,6 +40,9 @@ module "service" {
   env         = "staging"
   name        = "email"
   logs_bucket = data.terraform_remote_state.platform.outputs.logs_bucket
+
+  # Email in staging no longer keeps records.
+  records_table = false
 }
 
 output "jobs_queue" {
